manifest/v1alpha/report/validation_system_health_review.go

```diff
@@ -0,0 +1,88 @@
+package report
+
+import (
+	"time"
+
+	"github.com/pkg/errors"
+	"github.com/teambition/rrule-go"
+
+	"github.com/nobl9/nobl9-go/internal/validation"
+)
+
+var systemHealthReviewValidation = validation.New[SystemHealthReviewConfig](
+	validation.For(func(s SystemHealthReviewConfig) RowGroupBy { return s.RowGroupBy }).
+		WithName("rowGroupBy").
+		Required().
+		Rules(RowGroupByValidation()),
+	validation.ForSlice(func(s SystemHealthReviewConfig) []ColumnSpec { return s.Columns }).
+		WithName("columns").
+		Rules(validation.SliceMinLength[[]ColumnSpec](1)).
+		Rules(validation.SliceMaxLength[[]ColumnSpec](30)).
+		IncludeForEach(columnValidation),
+	validation.For(func(s SystemHealthReviewConfig) string { return s.TimeFrame.TimeZone }).
+		WithName("timeZone").
+		Required().
+		Rules(validation.NewSingleRule(func(v string) error {
+			if _, err := time.LoadLocation(v); err != nil {
+				return errors.Wrap(err, "not a valid time zone")
+			}
+			return nil
+		})),
```

Comment:
Why can't this be a part of the next step in validation that specifically checks `SystemHealthReviewTimeFrame`? Also, I don't know if it matters, but it makes more sense to me if we first check `s.TimeFrame` and then check its fields like `s.TimeFrame.TimeZone`.

Reply:
True, what do you think of the changes?

Reply:
I like them! Thanks.